### ISSUE TITLE
Crashes GTK4 Applications

### ISSUE THREAD

**Issue by rongcuid** [pop-os/gtk-theme#595]
## Describe the problem:

I am building a GTK4 application (reproduction here: [1]), and Pop theme causes a crash due to missing asset:

```
(hello-gtk4:1297239): Gdk-ERROR **: 12:18:13.975: Resource path /org/pop-os/themes/Pop/4.0/assets/check@2-symbolic.symbolic.png is not a valid image: The resource at “/org/pop-os/themes/Pop/4.0/assets/check@2-symbolic.symbolic.png” does not exist
fish: Job 1, './target/debug/hello-gtk4' terminated by signal SIGTRAP (Trace or breakpoint trap)
```

[1]: https://github.com/rongcuid/gtk4-rs-playground

## Steps to reproduce:
1. Clone https://github.com/rongcuid/gtk4-rs-playground
2. `cargo run`
3. Create a new TODO
4. Click on checkbox
5. Program crashes with above message
6. Rerun with `GTK_THEME=adwaita cargo run`, repeat steps
7. Program does not crash

**Distribution:**

```
PRETTY_NAME="Pop!_OS 22.04 LTS"
```

**Pop version:**

```
Installed: 5.5.1~1659710972~22.04~7ba5da8
```
```
Pop Gtk theme - Pop Gtk theme used by System76

          ID: org.gtk.Gtk3theme.Pop
         Ref: runtime/org.gtk.Gtk3theme.Pop/x86_64/3.22
        Arch: x86_64
      Branch: 3.22
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 1.1 MB

      Commit: 1eac06f6a58047fd6717a13f12b744ea59e6f9829fed02d856a5efb998b318a5
      Parent: bf0014cc176fe2a66eb69045bc3ed80663e09240dda1db97b5394ba3a501bea3
     Subject: add flathub.json (84e0033a)
        Date: 2020-11-17 21:00:45 +0000
```

**Comment by isantop** [comment]
So, neither I nor the QA team were able to recreate the issue with the version of the Pop theme currently released in 22.04. I did go ahead and update the theme to include the missing assets in the error message, so that should at least go away at this point.

**Comment by rongcuid** [comment]
I can't reproduce it either in another computer... I will see whether my original repo still fails once I get back.
